src/math/Vector3.js

```diff
@@ -1,22 +1,26 @@
 import { MathUtils } from './MathUtils.js';
 import { Quaternion } from './Quaternion.js';
 
-const _vector = new Vector3();
+let _vector;
```

Comment:
Hoisting does not work anymore, because Vector3 is not a function.
Vec3 instance will be assigned at the end of this file.